Neos.ContentRepository.Core/Classes/Factory/ContentRepositoryFactory.php

```diff
@@ -54,23 +69,41 @@ public function __construct(
         ProjectionsAndCatchUpHooksFactory $projectionsAndCatchUpHooksFactory,
         private readonly UserIdProviderInterface $userIdProvider,
         private readonly ClockInterface $clock,
+        SubscriptionStoreInterface $subscriptionStore,
     ) {
         $contentDimensionZookeeper = new ContentDimensionZookeeper($contentDimensionSource);
         $interDimensionalVariationGraph = new InterDimensionalVariationGraph(
             $contentDimensionSource,
             $contentDimensionZookeeper
         );
+
+
+        $eventNormalizer = new EventNormalizer();
         $this->projectionFactoryDependencies = new ProjectionFactoryDependencies(
             $contentRepositoryId,
-            $eventStore,
-            new EventNormalizer(),
+            $eventNormalizer,
             $nodeTypeManager,
             $contentDimensionSource,
             $contentDimensionZookeeper,
             $interDimensionalVariationGraph,
             new PropertyConverter($propertySerializer)
         );
         $this->projectionsAndCatchUpHooks = $projectionsAndCatchUpHooksFactory->build($this->projectionFactoryDependencies);
+        $subscribers = [];
+        foreach ($this->projectionsAndCatchUpHooks->projections as $projection) {
+            $subscribers[] = new Subscriber(
+                SubscriptionId::fromString(substr(strrchr($projection::class, '\\'), 1)),
+                SubscriptionGroup::fromString('default'),
+                RunMode::FROM_BEGINNING,
+                new ProjectionEventHandler(
+                    $projection,
+                    $this->projectionsAndCatchUpHooks->getCatchUpHookFactoryForProjection($projection)?->build($this->getOrBuild()),
```

Comment:
does that work or is it full recursion? :D

Reply:
probably not yet